### PR TITLE
Add support for Lenovo Vibe K5 variant with MSM8939 and HD panels (A6020a40 HW39)

### LINKED ISSUE #326: Doesn't boot with my Lenovo Vibe K5 (A6020a40)
I tried fastboot flash boot lk2nd.img. after reboot it doesn't boot up.

### PR DESCRIPTION
**Pull request by archeYR**
Add support for another variant of Lenovo Vibe K5 phone (should fix #326)

### CODE REVIEW

File: dts/msm8916/msm8939-qrd-wt82918.dts [hunk 1]
```diff
@@ -8,17 +8,7 @@
...
-		qcom,mdss_dsi_otm1901a_tm_1080p_video {
-			compatible = "wingtech,tm-otm1901a";
-		};
-		qcom,mdss_dsi_nt35596_1080p_video {
-			compatible = "wingtech,nt35595";
-		};
-	};
-};
\ No newline at end of file
+	model = "Lenovo Vibe K5 (Wingtech WT82918)";
```
Comment (TravMurav) [inline dts/msm8916/msm8939-qrd-wt82918.dts:11]: Is "Plus" dropped deliberately here?

File: dts/msm8916/msm8939-qrd-wt82918.dts [hunk 2]
```diff
@@ -8,17 +8,7 @@
...
+	compatible = "wingtech,wt82918", "qcom,msm8929", "lk2nd,device";
```
Comment (wonderfulShrineMaidenOfParadise) [inline dts/msm8916/msm8939-qrd-wt82918.dts:12]:
```suggestion
	compatible = "wingtech,wt82918", "qcom,msm8939", "lk2nd,device";
```

This is still msm8939, I assume?

File: dts/msm8916/wingtech-wt82918.dtsi [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (wonderfulShrineMaidenOfParadise) [inline dts/msm8916/wingtech-wt82918.dtsi:1]:
```suggestion
```

**Comment by TravMurav** [comment]
Thanks!

**Pull request merged**